src/toil/test/wdl/builtinTest.py

```diff
@@ -230,6 +246,22 @@ def testFn_Length(self):
         self.assertEqual(3, length(['a', 'b', 'c']))
         self.assertEqual(0, length([]))
 
+    def testFn_Zip(self):
+        """Test the wdl built-in functional equivalent of 'zip()'."""
+        xs = [1, 2, 3]
+        ys = ['a', 'b', 'c']
```

Comment:
Nit: `xs`, `val`, and `ys` may be a bit terse.  Consider renaming these so that they're more descriptive (maybe something including `right` and `left`?).